core/solidus_core.gemspec

```diff
@@ -35,6 +35,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'awesome_nested_set', '~> 3.3'
   s.add_dependency 'cancancan', ['>= 2.2', '< 4.0']
   s.add_dependency 'carmen', '~> 1.1.0'
+  s.add_dependency 'concurrent-ruby', '< 1.3.5'
```

Comment:
Can we add a comment to the Rails issue?

Reply:
Added.